makisu/makisu.yaml

```diff
@@ -2,43 +2,56 @@ apiVersion: tekton.dev/v1beta1
 kind: Task
 metadata:
   name: makisu
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tags: image-build
 spec:
+  description: >-
+    This Task builds source into a container image using uber’s
+    makisu tool.
+
+    Makisu is a fast and flexible Docker image build tool designed for unprivileged
+    containerized environments such as Mesos or Kubernetes. makisu uses a registry
+    configuration which should be stored as a secret in Kubernetes.
+
   params:
-  - name: CONTEXTPATH
-    description: Path to the build context.
-    default: /workspace
-  - name: PUSH_REGISTRY
-    description: Registry to push image to.
-    default: index.docker.io
-  - name: REGISTRY_SECRET
-    description: Secret containing information about the used regsitry.
-    default: docker-registry-config
+    - name: CONTEXTPATH
```

Comment:
this whitespace change is unrelated to the patch. could you please ensure that only necessary changes are made to the yaml files 🙏 👼 ?